histomicstk/segmentation/label/isbfcpp.h

```diff
@@ -0,0 +1,42 @@
+/*
+C++ version of ISBF for TraceBounds
+*/
+#ifndef isbfcpp_h
+#define isbfcpp_h
+
+#include <iostream>
+#include <string>
+#include <vector>
+
+struct node
+{
+    int val_x;
+    int val_y;
+    node *next;
+};
```

Comment:
Use `std::list` instead of this. See [here](http://www.cplusplus.com/reference/list/list/push_back/)